model/patch_lifecycle.go

```diff
@@ -875,13 +875,22 @@ func getLoadProjectOptsForPatch(p *patch.Patch, githubOauthToken string) (*Proje
 		hash = p.GithubMergeData.HeadSHA
 	}
 
+	manifestID := ""
+	baseVersion, err := VersionFindOne(BaseVersionByProjectIdAndRevision(p.Project, p.Githash))
+	if err == nil && baseVersion != nil {
```

Comment:
Looking at the implementation, we should return the error. It separates out no results found to return nil rather than an error- so if it encounters an error it is from something else.